apps/chat/src/utils/app/data/storages/api-storage.ts

```diff
@@ -111,8 +112,30 @@ export class ApiStorage implements DialStorage {
     return this._conversationApiStorage.getEntity(info);
   }
 
-  createConversation(conversation: Conversation): Observable<void> {
-    return this._conversationApiStorage.createEntity(conversation);
+  createConversation(
+    conversation: Conversation,
+  ): Observable<ConversationInfo | null> {
+    return this._conversationApiStorage.createEntity(conversation).pipe(
+      catchError(() => {
+        return this.getConversations(conversation.folderId).pipe(
```

Comment:
Looks like this logic should be inside conversation api storage but not in API storage